src/dialogs/config-flow/step-flow-pick-handler.ts

```diff
@@ -144,6 +141,32 @@ class StepFlowPickHandler extends LitElement {
     `;
   }
 
+  private _renderRow(handler: HandlerObj) {
+    return html`
+      <paper-icon-item @click=${this._handlerPicked} .handler=${handler}>
```

Comment:
Should we just move this on over to MWC while we are here?

Reply:
Done.